source/agora/network/NetworkClient.d

```diff
@@ -52,8 +69,7 @@ class NetworkClient
     /// Ban manager
     private BanManager banman;
 
-    /// API client to the node
-    private API api;
```

Comment:
I don't see how this could work. Node is the business code of the server. `API` is the public interface, which can be used to instantiate a client. A `NetworkClient` is just an instance owned by `NetworkManager` that uses an `API` in order to perform queries.

If you store a `Node` here, you basically store another node in a node, instead of storing a client.

Reply:
You're right, it's still not finished - api substituted by node just for compilation check